src/modules/Nfs.rb

```diff
@@ -500,17 +503,15 @@ def Write
     # Summary()
     # @return Html formatted configuration summary
     def Summary
+      log.info("Entries: #{@nfs_entries}")
```

Comment:
I'm not familiar with NFS much, is it possible that the entries could possibly contain some secret data (like passwords)? This could potentially leak some private values when attaching the log to bugzilla.

E.g. for URLs we use `URL.HidePassword` when logging any URL. Is it safe for the NFS entries?

Reply:
I just checked; there are articles on superuser.com about passwords in the mount options (although I can't find a reference to that in any of the NFS man pages - but whatever), and the mount options are definitely in that log output.

Thus, simply removing that log line; it wasn't there in the old code, so we won't need it in the new one either.